src/parse-result.js

```diff
@@ -0,0 +1,73 @@
+/*
+ * Parse result-specific refract elements.
+ *
+ * General structure:
+ *
+ * + ParseResult
+ *   + Annotation
+ */
+
+import apiDescription from 'minim-api-description';
+
+export function namespace(options) {
+  const minim = options.base;
+  const ArrayElement = minim.getElementClass('array');
+  const StringElement = minim.getElementClass('string');
+
+  class ParseResult extends ArrayElement {
+    constructor() {
+      super(...arguments);
+      this.element = 'parseResult';
+    }
+
+    get api() {
+      return this.children(item => item.classes.contains('api')).first();
```

Comment:
Ah, so, if we hadn't done `use()` for `api description` in this file, this line wouldn't return the `Category` instance but since we did, this is now good to go?